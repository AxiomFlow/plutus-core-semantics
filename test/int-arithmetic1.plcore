(builtin addInt ((builtin multiplyInt (3 3)) (builtin multiplyInt (4 3))))
