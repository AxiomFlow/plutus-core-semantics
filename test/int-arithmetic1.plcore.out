<T> <k> 21 </k> </T>
